sandy/core/cov.py

```diff
@@ -509,29 +509,37 @@ def invert(self, rows=None):
         M_inv = M_inv.reindex(index=index, columns=columns).fillna(0)
         return self.__class__(M_inv)
 
-    def sampling(self, nsmp, seed=None, rows=None, pdf='normal',
-                 tolerance=None, threshold=None):
+    def sampling(self, nsmp, seed=None, rows=None, mean=1, pdf='normal',
+                 tolerance=None, threshold=None, relative=True):

```

Comment:
As mentioned in another PR. I am not sure how useful this `threshold` card can be

Reply:
It will be kept in mind. But we will remove it!

Reply:
It will be kept in mind. But we will remove it!